app/src/test/java/com/eis/geoCalendar/app/MockEventDatabase.java

```diff
@@ -49,4 +51,22 @@ public boolean removeEvent(@NonNull E event) {
     public ArrayList<E> getSavedEvents() {
         return events;
     }
+
+    /**
+     * Retrieves presence of event.
+     *
+     * @param event The event to find.
+     * @return {@code true} if the event is parsable and present, {@code false} otherwise.
+     */
+    @Override
+    public boolean contains(@NonNull E event) {
+        return events.contains(event);
+    }
+    /**
+     * @return The number of events in the database.
+     */
+    @Override
+    public int count(){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['{' is not preceded with whitespace.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4822079)